openfoam-interface/internal-interface/wrapper/simphonyInterface.C

```diff
@@ -945,6 +943,9 @@ void foam_modifyNumerics(std::string name, std::string fvSch, std::string fvSol,
         cDIS()
     );
 
+    TimeMod* t = (TimeMod*)(runTimes[name]);
```

Comment:
Regardless how `TimeMod` is used, it looks like it can be possibly replaced by just using the *public* `Time::read`.  This method looks like it does what you need it to do, right?